votes/0639-propose-a-release-note-guide.md

```diff
@@ -0,0 +1,76 @@
+# Propose a Release Notes Language Style Guide
+
+## Background
+
+A concise release note can clearly and accurately deliver to users how your PR can make a difference. Your release note written in a PR will be presented in [docs.pingcap.com](https://docs.pingcap.com/tidb/stable/release-notes) as a part of the TiDB documentation.
+
+I hereby propose a release notes language style guide that briefly explains what a quality release note looks like, provides examples, and aims to help you write quality release notes.
+
+## What a quality release note looks like
+
+A high-quality release note has the following merits:
+
+- Clear in type
+- Adequate and clear in meaning
+- User perspective
+
+A release note with a distinguishable type can help users quickly identify the nature or goal of your PR change. Other teams will also benefit from it.
+
+Depending on what your PR changes, you can refer to one of the following release note types:
+
+- Compatibility change
+- Improvement or Feature enhancement
+- Bug fix
+
+## Compatibility change
+
+A compatibility change note means:
+
+- Your PR adds, removes, or modifies one or more configuration items or system variables.
+- Your PR modifies the default value of a configuration item or system variable.
+
+For this type of note, you should clearly and adequately state the following aspects:
+
+- The previous code behavior, configuration item, or default value.
+- The new code behavior, configuration item, or default value since the new version.
+
+Note that the object of the change should be user-perceivable. If the changed configuration item or system variable is not supposed to be exposed to users, do not include it in your release notes.
+
+**Examples:**
+
+| **Not recommended**                                          | **Clear in type** | **Adequate and clear in meaning** | **User perspective** | Recommended                                                  |
+| ------------------------------------------------------------ | ----------------- | --------------------------------- | -------------------- | ------------------------------------------------------------ |
+| copr: cast invalid utf8 string to real bug fix               | ❌                 | ❌                                 | ❌                    | Before v4.0.16, when TiDB converts an illegal UTF-8 string to a Real type, an error is reported directly. Starting from v4.0.16, TiDB processes the conversion according to the legal UTF-8 prefix in the string |
+| sink: fix kafka max message size inaccurate issue            | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink max-message-bytes to 1 MB to prevent TiCDC from sending too large messages to Kafka clusters |
```

Comment:
this recommended stating does not include "The previous code behavior, configuration item, or default value.". I guess because the PR author can't infer the previous behaviour from given information?

Reply:
```suggestion
| sink: fix kafka max message size inaccurate issue            | ❌                 | ❌                                 | ❌                    | Change the default value of Kafka Sink max-message-bytes from 512 MB to 1 MB to prevent TiCDC from sending too large messages to Kafka clusters |
```